### ISSUE TITLE
Can I avoid "#include <iostream>" being preprocessed ?

### ISSUE THREAD

**Issue by aachenhang** [mjspncr/lzz3#24]
If I don't want to preprocess the "#include<iostream>" and instead want to simply copy it to the output file, what can I do?

**Comment by mjspncr** [comment]
Typically you put your include directives in 
```
#hdr
copied verbatim to top of header file
#end
```
or
```
#src
copied verbatim to top of cpp file
#end
```
Note you can put these lines anywhere in the lzz file. Include directives outside these blocks are processed by lzz -- but just to pull in macros. You might want to do this if you have macros that expand to function or class definitions for example.

**Comment by brokenbeta** [comment]
Since it seems like this is a very common pattern, would you be open to an abbreviated syntax that only takes up a single line?

    #src #include "foo.h"
    #src include "foo.h"
    #srcinclude "foo.h"

or similar?

**Comment by mjspncr** [comment]
Yeah, kind of neat. Could say anything after ```#src``` goes to source file .. last 2 options could just be special case of the first.
